nscplugin/src/main/scala/scala/scalanative/nscplugin/NirGenStat.scala

```diff
@@ -588,6 +591,52 @@ trait NirGenStat[G <: nsc.Global with Singleton] { self: NirGenPhase[G] =>
           ()
       }
 
+    def genJavaDefaultMethodBody(dd: DefDef): Seq[nir.Inst] = {
+      val fresh = Fresh()
+      val buf   = new ExprBuffer()(fresh)
+
+      implicit val pos: nir.Position = dd.pos
+
+      val sym                    = dd.symbol
+      val forwardeeClassFullName = sym.owner.fullName + "$class"
```

Comment:
I'm not a native English speaker, but to me the forwardee would be the default method, since we forward the default method *to* the `$class`'s method. So I find that name misleading. Perhaps call it `targetClassFullName`, or even `implClassFullName` since the `$class` is internally known as an "impl class". The method sym can also be named `targetMethodSym` or `implMethodSym`.

Reply:
Changed "forwardee" to "impl" as suggested. Thank you.